pkg/i18n/english.go

```diff
@@ -1323,6 +1324,7 @@ func EnglishTranslationSet() TranslationSet {
 			ForceCheckoutBranch:               "Force checkout branch",
 			DeleteBranch:                      "Delete branch",
 			Merge:                             "Merge",
+			Rebase:                            "Rebasing",
```

Comment:
My apologies: looks like Actions is the wrong place to put this (it's using present tense and everything else is tenseless). In fact, we already have a `RebasingStatus` to use! So let's just use the existing one